recipes/backpack/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "backpack" %}
+{% set version = "0.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "0162cf7b34c810ba4ddbbd32a1e5e804ef96fcf2fea5ce2848aa4950770d3893" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - simplejson
+  run:
+    - python
+    - simplejson
+
+test:
+  imports:
+    - backpack
+    - backpack._utils
+    - backpack.collections
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/sdispater/backpack
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
+  summary: Useful utilities for Python.
+  description: "Useful utilities for Python. Supports Python 2.7+ and 3.2+."
+  doc_url: https://github.com/sdispater/backpack
+  dev_url: https://github.com/sdispater/backpack
+
+extra:
+  recipe-maintainers:
```

Comment:
Why were the R maintainers added to this Python recipe? Is it a dependency for an R package?